src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java

```diff
@@ -94,6 +94,17 @@ public void upload(PdfDoc pdfDoc) {
         });
     }
 
+    public void deleteReport(String reportPath) {
```

Comment:
Can we add unit test cases covering this method including exception please?Everything else looks good to me :)

Reply:
done